### ISSUE TITLE
LODCM Energy Motors

### ISSUE THREAD

**Issue by ZLLentz** [pcdshub/pcdsdevices#704]
## Expected Behavior

`LODCM.energy` pseudomotor
- sets to four LODCM motors using calculations, see old formulation at https://github.com/pcdshub/old-hutch-configs/blob/18c7769d66a647ab558ad7ebc34d019fc98fbf48/xpp/blinst/lom.py#L1197
- reads from one motor using calculations, see old formulation at https://github.com/pcdshub/old-hutch-configs/blob/18c7769d66a647ab558ad7ebc34d019fc98fbf48/xpp/blinst/lom.py#L1121

## Current Behavior

Does not exist in the new python

## Possible Solution

Investigate calculations, expand `pcdscalc` may be needed.

## Context

This would allow scanning the LODCM energy

**Comment by cristinasewell** [comment]
Are we not using the `T2C:REF` and `T2Si:REF` on purpose here - are they usually the same as the `T1C:REF` and `T1Si:REF`?

**Comment by ZLLentz** [comment]
That's really cryptic... I have no idea. It looks like an error? But maybe it never causes an issue because they are generally the same?

**Comment by cristinasewell** [comment]
This PR is on my list of Issues now, and I am trying to figure out what else still needs to be done here.

I think we covered most of this PR (or at least a portion of it) in #715 and #766, as well as in pcdshub/pcdscalc#9.
@ZryletTC do you know if the scientists got a chance to test the LODMC energy stuff at all?

I also tagged Diling a while ago in this PR (https://github.com/pcdshub/pcdsdevices/issues/763#issuecomment-785360614) to ask for any other functions we might need in LODCM but so far we did not  get anything.

**Comment by ZryletTC** [comment]
@cristinasewell Yeah, they've tested but it's been awhile so I'm not sure if either they or I remember much about what still needs to be done without revisiting it more recently. The only thing I really remember is that I think there were a bunch of subscription errors popping up so we ended up disabling at least the XCS LODCM. I'll see if we can test it out again this week so I can give you a better picture of the current status.
